app/Http/Controllers/TaskCategoriesController.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use Illuminate\Support\Facades\Validator;
+
+use App\Models\TaskCategory;
```

Comment:
`use` statements should be written in ascending order of their length. So the above should be...

```php
use Illuminate\Http\Request;
use App\Models\TaskCategory;
use Illuminate\Support\Facades\Validator;
```
There should be no line breaks between `use` statements.

These comments and any other comments I make in the PR should be addressed in all the files and not just this file.

Reply:
@abbasali ok sir